share/rocm/cmake/ROCMCreatePackage.cmake

```diff
@@ -28,6 +28,16 @@ macro(rocm_create_package)
         set( CPACK_SET_DESTDIR ON CACHE BOOL "Boolean toggle to make CPack use DESTDIR mechanism when packaging" )
     endif()
 
+    rocm_set_os_id(_os_id)
+    rocm_read_os_release("VERSION_ID" _version_id)
+    
+    #only set CPACK_SYSTEM_NAME for AMD supported OSes
+    if (_os_id_centos OR _os_is_rhel)
+        STRING(CONCAT CPACK_SYSTEM_NAME "el" ${_version_id} ".x86_64")
+    elseif(_os_id_ubuntu OR _os_is_sles)
+        STRING(CONCAT CPACK_SYSTEM_NAME ${_os_id} "-"${_version_id} ".amd64")
```

Comment:
These should use an underscore instead of `.`. So it should be `_amd64` and `_x86_64`. This is usually common for package names(at least in ubuntu):

```
$ ls /var/cache/apt/archives/  | head
clang-8_1%3a8-3~ubuntu18.04.1_amd64.deb
libclang1-8_1%3a8-3~ubuntu18.04.1_amd64.deb
libclang-common-8-dev_1%3a8-3~ubuntu18.04.1_amd64.deb
libomp5-8_1%3a8-3~ubuntu18.04.1_amd64.deb
libomp-8-dev_1%3a8-3~ubuntu18.04.1_amd64.deb
llvm-8_1%3a8-3~ubuntu18.04.1_amd64.deb
llvm-8-dev_1%3a8-3~ubuntu18.04.1_amd64.deb
llvm-8-runtime_1%3a8-3~ubuntu18.04.1_amd64.deb
```

Reply:
For Centos, dot seems more common. https://rpmfind.net/linux/rpm2html/search.php?query=gcc&submit=Search+...

I am going to search a little more. 
I went with amd64 for sles and ubuntu as they use that more often. redhat seems to prefere x86_64.

Reply:
Might have to do something distinct for each OS.

Reply:
Maybe it is a deb versus RPM convention for period or underscore.

Reply:
Yea, maybe RPMs use a dot and debians use an underscore.